released/packages/coq-mi-cho-coq/coq-mi-cho-coq.0.1/url

```diff
@@ -0,0 +1,2 @@
+http: "https://gitlab.com/nomadic-labs/mi-cho-coq/-/archive/version-0.1/mi-cho-coq-version-0.1.tar.gz"
```

Comment:
Should be in the `opam` file (see another package as an example, file to remove).

Reply:
Maybe you want to update your blog post then: http://coq-blog.clarus.me/make-a-coq-package.html

Reply:
Arggggggggg! Yes, I should do that, I did not know this was out of date :see_no_evil:

Reply:
Thanks

Reply:
I have put the urls in the opam files and removed the url files.